scalameta/contrib/src/main/scala/scala/meta/contrib/instances/StatementsInstances.scala

```diff
@@ -0,0 +1,117 @@
+package scala.meta.contrib.instances
+
+import scala.collection.immutable.Seq
+import scala.meta._
+import scala.meta.contrib._
+import scala.meta.contrib.implicits.Statements
+
+trait StatementsInstances {
+  implicit val termBlockStats: Statements[Term.Block] = new Statements[Term.Block] {
+    override def getStats(block: Term.Block): Seq[Stat] = block.stats
+
+    override def replaceStats(block: Term.Block, newStats: Seq[Stat]): Term.Block =
+      block.copy(stats = newStats)
+  }
+
+  implicit val packageStats: Statements[Pkg] = new Statements[Pkg] {
+    override def getStats(pkg: Pkg): Seq[Stat] = pkg.stats
```

Comment:
Crazy idea, handle nested Pkg?

```scala
@ clangPrint(source"""package a
           package b
           class C""".structure)
res2: String = """
Source(Seq(Pkg(Term.Name("a"),
               Seq(Pkg(Term.Name("b"),
                       Seq(Defn.Class(Nil, Type.Name("C"), Nil, Ctor.Primary(Nil, Ctor.Ref.Name("this"), Nil),
                                      Template(Nil, Nil, Term.Param(Nil, Name.Anonymous(), None, None), None))))))))
"""
```

That is, if `stats` has only a single `Pkg`, then concat with the stats of that Pkg. If so, `replaceStats` should handle that case too, which to wrap the tail into the `head: Pkg` element.